Formula/m/mpv.rb

```diff
@@ -27,7 +27,6 @@ class Mpv < Formula
   depends_on "libass"
   depends_on "libbluray"
   depends_on "libplacebo"
-  depends_on "libsamplerate"
```

Comment:
Unused on macOS?

Reply:
Didn't see any linkage in existing bottles. Also didn't find any specific reference in source code to `libsamplerate`